packages/flutter/test/widgets/list_wheel_scroll_view_test.dart

```diff
@@ -68,6 +68,143 @@ void main() {
     });
   });
 
+  group('infinite scrolling', () {
```

Comment:
@xster I think this will be outside the scope of this PR, but we should consider adding performance testing for the scrolling behavior on this widget in the future.

Reply:
I think a full on black-box performance test is definitely outside the scope of this PR. But I definitely hope that we can unit test that the performance-allowing behaviors are respected here. 

e.g.: have a builder delegate and the parent is never rebuilt (i.e. the builder function never changes) and scroll around and assert each index is never called to build twice. 

Have a list delegate, have one of the child in the list be a builder. Make sure that builder isn't invoked more than once when scrolling around.

Reply:
That's actually a very good test. I have added that.

Reply:
Were these tests added? Also check that scrolled-off render objects are not attached.

Reply:
yup, a test has been added to check that builder wont be called twice.
I have also just added a test to check that only visible children are attached.